ushahidi/platform-client-mzima
# [Outreachy Task Submission]: fix: make bulk actions more accessible to keyboard users

This pr fixes issue [#4895](https://github.com/ushahidi/platform/issues/4895). The issue focused on the accessibility issues related to bulk actions. This pr addresses that issue in the following ways:

- Aria label added to bulk actions button with the text "Enable bulk actions" when bulk actions is not yet enabled and "disable bulk actions" when it is enabled.
- Focus moves programmatically to the posts when bulk actions is enabled.
- Aria-labels have been added to the input fields with the text "Select post: {post's title}".
- Individual post actions are disabled when bulk actions is enabled.
- Tabbing through the posts should move from one checkbox to the next.

To test the fix: 

- With a screen reader enabled, focus on the bulk actions button. Screen reader should describe button as "Enable bulk actions".
- If you activate the button, screen reader should now describe it as "Disable bulk actions".
- When button is activated, focus should automatically move to the posts.
- While focused on a post's checkbox, checkbox should be described with the post's title or its content if a title is not available.
- Tabbing through the posts should move from one checkbox to the next.
